### ISSUE TITLE
PyQt5.QtWebEngineWidgets missing from install

### ISSUE THREAD

**Issue by rglanz** [cortex-lab/phy#1021]
I installed using the developer instructions and got this error.

`ModuleNotFoundError: No module named 'PyQt5.QtWebEngineWidgets'`

Looks like it got moved to a separate package. Running `pip install PyQtWebEngine` fixed the issue.

**Comment by rossant** [comment]
Yes this is a known issue: https://github.com/cortex-lab/phy#dealing-with-the-error-modulenotfounderror-no-module-named-pyqt5qtwebenginewidget
The problem is that it looks like this command should only run on particular environments (combination of OS and Python distribution), and not others otherwise it fails, and it's unclear to me which!

**Comment by NixPlayer** [comment]
just run in a terminal:
pip install PyQtWebEngine

**Comment by rossant** [comment]
Fixed in phy 2.1. Please reopen if this still happens.

**Comment by rglanz-rc** [comment]
Nice! Impressed you addressed such a backlogged issue

![image](https://github.com/user-attachments/assets/9999f795-64aa-46ba-b152-875d4f903438)
